FlashLFQ/Bayesian Protein Quantification/ProteinQuantificationEngine.cs

```diff
@@ -157,15 +157,15 @@ public void Run()
                             result.CalculatePosteriorErrorProbability(new double[] { nullHyp });
                         }
 
-                        protein.conditionToQuantificationResults.Add(condition, result);
+                        protein.ConditionToQuantificationResults.Add(condition, result);
```

Comment:
PR looks fine. there is a merge conflict but I'll re-approve when that's fixed, lmk